modules/navier_stokes/examples/solidification/galium_melting.i

```diff
@@ -0,0 +1,369 @@
+##########################################################
+# Simulation of Galium Melting Experiment
+# Ref: Gau, C., & Viskanta, R. (1986). Melting and solidification of a pure metal on a vertical wall.
```

Comment:
It would be cool to have this reference in `navier_stokes.bib` but I don't have a brilliant idea for where to put a reference for this in our `.md` files